### ISSUE TITLE
Unable to add image + text to HTML Mesh

### ISSUE THREAD

**Issue by user080975** [mrdoob/three.js#25655]
### Description

Hi,

I'm trying to embed some HTML containing both images and text into HTML Mesh and adding the group to a scene. But so far only texts appear in the mesh, not images. Does HTML Mesh support embedding images? 

Thank You!

### Reproduction steps

1. Create an HTML element containing images and text
2. Create an HTML mesh object using new HTMLMesh()
3. Add mesh to scene

### Code

```js
const group = new InteractiveGroup( renderer, camera );
scene.add( group );
const test = document.querySelector('#test')
const htmlMesh = new HTMLMesh(test)
htmlMesh.scale.setScalar(10)
htmlMesh.position.set(-2, 0, 0)
group.add(htmlMesh)
```

### Live example

* [jsfiddle-dev](https://jsfiddle.net/jcy089/vhu8kmzt/4/)

### Screenshots

<img width="490" alt="2023-03-11 at 11 48 41" src="https://user-images.githubusercontent.com/54892636/224480022-5da833e3-6943-4092-899b-749df58e900b.png">

### Version

r137

### Device
Mac

### Browser
Safari

### OS
Mac OS

**Comment by LeviPesin** [comment]
Related: https://github.com/mrdoob/three.js/pull/24043

**Comment by user080975** [comment]
In the pull request I'm seeing that the image `https://i.imgur.com/jf4lNex.png` has been successfully rendered into the HTML Mesh. 

My issue is that I'm not even seeing my image rendered at all in my example JS Fiddle. 

So does HTML Mesh support images just with some slight bugs or not at all?

**Comment by Mugen87** [comment]
No, there is no support for images.

I also wonder if the support is actually required since you could easily load the image as a normal texture and add it to a plane mesh. The mesh itself could be added as a child to your interactive group.

**Comment by Mugen87** [comment]
I understand. Maybe we can revisit #24043 and try to fix the open points.

**Comment by user080975** [comment]
If the image loading is fixed, will it be possible to use the new HTMLMesh.js with r137 as an add-on? The project I'm working on is using r137 and I think a lot of other elements might brake if there was a sudden migration.

**Comment by Mugen87** [comment]
It _should_ work although there is no guarantee. We do not support such scenarios.

**Comment by user080975** [comment]
If the InteractiveGroup for the HTML Mesh isn't passed into the render loop, is there any method to manually update the HTML Mesh? Something like needsUpdate = true like with textures?

Since I'm not updating it often, I'd like to manually call update for new changes to the HTML.

**Comment by Mugen87** [comment]
You can manually update the HTML mesh's internal texture like so:
```js
htmlMesh.material.map.update();
```
This will trigger the `html2canvas()` routine that redraws its canvas based in the latest DOM.

**Comment by user080975** [comment]
Thank you, this worked great! I see that there's a new pull request, does this mean the issue has been fixed? If so, how can I download the fixed version of htmlMesh.js?

**Comment by Mugen87** [comment]
>  does this mean the issue has been fixed

Not yet. The PR has to be merged first. 

After that, the latest `dev` version can be found here: https://github.com/mrdoob/three.js/blob/dev/examples/jsm/interactive/HTMLMesh.js

**Comment by user080975** [comment]
Great! Is there an approximate date when the PR might be merged?

**Comment by Mugen87** [comment]
Done!

**Comment by user080975** [comment]
I just tried the new HTMLMesh.js and it does render images now. But for some reason texts and other HTML aren't rendered anymore. I've attached screenshots showing the result from the current production version of HTMLMesh.js and the new fix:

https://imgur.com/a/QdMOuAw

Is it possible to keep both text and images together in the canvas?
